app/exceptions/handlers.py

```
@@ -0,0 +1,85 @@
+
+generic_exception_handler_status_code = status.HTTP_503_SERVICE_UNAVAILABLE
+
+
+def generic_exception_handler(request: Request, exc: Exception) -> dict[str, Any]:
+    path = request.url.path
+    method = getattr(request, "method", None) or request.scope.get("method") or "HTTP"
+    exc_type = type(exc).__name__
+    status_code = generic_exception_handler_status_code
+
+    logger.exception(
+        f"[UNHANDLED_EXCEPTION] {exc_type} occurred | Path: {path} | Method: {method} | Status: {status_code}"
```

Pylint (3.3.7)

[convention] Line too long (110/100)

(C0301)